README.fr.md

```diff
@@ -65,9 +65,10 @@ Si vous voulez une publication continue (rolling release) avec les dernières no
 
 ## Installation automatisée
 
-* [![Docker](https://www.docker.com/sites/default/files/horizontal.png)](./Docker/)
+* [![Docker](./docs/img/docker-horizontal.png)](./Docker/)
```

Comment:
Thanks for the fix. I think I would rather like to keep hot-linking to an image from Docker.com, to ease any logo rights issue

Reply:
Both fine with me. I removed the image from the PR as well.